rails/app/services/google_analytics.rb

```diff
@@ -0,0 +1,35 @@
+require 'google/api_client'
+require 'date'
+class GoogleAnalytics
+  def initialize
+    @client  = Google::APIClient.new(:application_name => ENV["GA_APP_NAME"], :application_version => '1.0')
+    key_file = File.join('config', 'analytic_cert.p12')
+    scope    = "https://www.googleapis.com/auth/analytics.readonly"
+    key      = Google::APIClient::PKCS12.load_key(key_file, "notasecret")
+    service_account = Google::APIClient::JWTAsserter.new(ENV["GA_EMAIL"], scope, key)
+    @client.authorization = service_account.authorize
+    @analytics = @client.discovered_api('analytics', 'v3')
+  end
+
+  def visitors(startDate, endDate)
+    results = @client.execute(:api_method => @analytics.data.ga.get, :parameters => {
+      'ids'         => ENV['GA_ID'],
+      'start-date'  => startDate,
+      'end-date'    => endDate,
+      'metrics'     => "ga:visits",
+      'dimensions'  => "ga:year,ga:month,ga:day",
+      'sort'        => "ga:year,ga:month,ga:day"
+    })
+    if results.error?
+      puts results.error_message
```

Comment:
This will probably never be seen. If you want visibility I'd either raise a 500 or send an email to yourself, log to Airbrake, or whatever.